code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -306,6 +306,20 @@
 					msg += "[t_He] [t_is] flushed and wheezing.\n"
 				if (bodytemperature < dna.species.bodytemp_cold_damage_limit)
 					msg += "[t_He] [t_is] shivering.\n"
+				if (HAS_TRAIT(src, TRAIT_PACIFISM))
+					msg += "[t_He] do[p_es()]n't look like [p_they()] would hurt a fly.\n"
+				if (HAS_TRAIT(src, TRAIT_CLUMSY))
+					msg += "[t_He] look[p_s()] like a clutz.\n"
+				if (HAS_TRAIT(src, TRAIT_GRABWEAKNESS))
+					msg += "[t_He] seem[p_s()] like a pushover.\n"
+				if (HAS_TRAIT(src, TRAIT_SKITTISH))
+					msg += "[t_He] [t_is] a bit skittish.\n"
+				if (HAS_TRAIT(src, TRAIT_ANXIOUS))
+					msg += "[t_He] [t_has] a nervous look.\n"
+				if (HAS_TRAIT(src, TRAIT_SOFTSPOKEN))
+					msg += "[t_He] seem[p_s()] shy.\n"
+				if (HAS_TRAIT(src, TRAIT_FRIENDLY))
+					msg += "[t_He] [t_has] a friendly look.\n"
```

Comment:
All these newlines are going to make this already long examine block even longer. I'd prefer if the quirks were listed in a single line, like this:
```
var/list/empath_quirks_message = list()
if (HAS_TRAIT(src, TRAIT_PACIFISM))
	empath_quirks_message += "peaceful"
if (HAS_TRAIT(src, TRAIT_CLUMSY))
	empath_quirks_message += "klutzy"
. . . . 
msg += "[t_He] also seem[p_s()] quite [english_list(empath_quirks_message)].\n"
```

Reply:
You could also do a mix of mine and Cobbie's requests. I'm not a huge fan of long columns of `if` statements.